cli/flags.rs

```diff
@@ -150,12 +168,21 @@ pub fn parse_flags(matches: ArgMatches) -> DenoFlags {
   if matches.is_present("allow-read") {
     flags.allow_read = true;
   }
+  if let Some(read_wl) = matches.values_of("read-wl") {
+    flags.read_whitelist = read_wl.map(|s| s.to_string()).collect();
+  }
   if matches.is_present("allow-write") {
     flags.allow_write = true;
   }
+  if let Some(write_wl) = matches.values_of("write-wl") {
+    flags.write_whitelist = write_wl.map(|s| s.to_string()).collect();
+  }
   if matches.is_present("allow-net") {
     flags.allow_net = true;
   }
+  if let Some(net_wl) = matches.values_of("net-wl") {
```

Comment:
I'm not sure about the actual naming of these flags.

In wasmtime, they have `--dir` which is close to these whitelist flags. I wonder if we should adopt that.

In any case, I think this is fine for a first pass with the understanding that we'd want to simplify this at some point.

Reply:
Naming wise why not reuse the original, so that --allow-net is equivalent to --allow-net=*.

Reply:
I was pondering it for some time now and @hayd's idea seems pretty ergonomic, but... consider that if there's a whitelist, people will probably want a blacklist as well. Surely this would be solvable by adding some special syntax (maybe `-`?, eg. `--allow-net=127.0.0.1,-0.0.0.0,`). But then again that would add some unneeded cognitive overhead and magic... Ideas? 

CC @rsp might be interested

Reply:
Semantically using - under --allow-net to signify blacklist is a bit of a stretch. Also imo one char difference is too small.

Another flag for --reject-net or something?

Reply:
`--no-net` could also work for blacklisting (consistent with other flags e.g. `--no-prompt`)
ie. `--no-net=facebook.com` and `--no-read=/etc/passwd`

Reply:
Going to go with @hayd suggestion on this one and use `--allow-x=`.